### PR TITLE
Rename light-controller to motion-led-controller

### PR DESCRIPTION
**Pull request by manohar-narkhede-imgtec**
Signed-off-by: Manohar Narkhede Manohar.Narkhede@imgtec.com

### CODE REVIEW

File: motion-led-controller/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Feeds for CreatorKit packages.
+
+Package           | Description
+:---------------- | -----------------------------
+button-gateway    | Button gateway is an application running on Ci40 which acts as a gateway for MikroE boards.
+button-led-controller    | Button-Led controller is an application for controlling LED on Ci40 as per button-press on Clicker boards.
+device-manager    | Device manager is an application for provisioning devices with access details for FlowCloud.
+libflow           | Binary package for FlowLibraries.
+motion-led--controller  | Motion Led controller is an application running on Ci40 which controls the LED based on the motion sensor events from MikroE board.
```
Comment (nikhil-zinjurde-imgtec) [inline motion-led-controller/README.md]: motion-led-controller

File: motion-led-controller/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+device-manager    | Device manager is an application for provisioning devices with access details for FlowCloud.
+libflow           | Binary package for FlowLibraries.
+motion-led-controller  | Motion Led controller is an application running on Ci40 which controls the LED based on the motion sensor events from MikroE board.
+webscripts        | Webscripts to provision Gateway and Constrained devices.
+
+Command                                         | Description
+:---------------------------------------------- | :---------------------------------------
+```$ ./scripts/feeds/install device-manager```  | Install only the package device-manager
+
+
```
Comment (nikhil-zinjurde-imgtec) [inline motion-led-controller/README.md]: Why are there 2 different README.md files? there should not be a motion-led-controller/README.md in openwrt-ckt-feeds

File: README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ button-gateway    | Button gateway is an application running on Ci40 which acts
 button-led-controller    | Button-Led controller is an application for controlling LED on Ci40 as per button-press on Clicker boards.
 device-manager    | Device manager is an application for provisioning devices with access details for FlowCloud.
 libflow           | Binary package for FlowLibraries.
-light-controller  | Light controller is an application running on Ci40 which controls the LED based on the motion sensor events from MikroE board.
+motion-led--controller  | Motion Led controller is an application running on Ci40 which controls the LED based on the motion sensor events from MikroE board.
```
Comment (nikhil-zinjurde-imgtec) [inline README.md]: motion-led-controller

File: motion-led-controller/Makefile [hunk 4]
```diff
@@ -38,14 +38,14 @@ define Build/Clean
...
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/set_led.sh $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/light_controller_continuous.sh $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/motion_led_controller_continuous.sh $(1)/usr/bin
 	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/light_controller_appd $(1)/etc/init.d/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/light_controller_continuousd $(1)/etc/init.d/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/motion_led_controller_appd $(1)/etc/init.d/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/files/motion_led_controller_continuousd $(1)/etc/init.d/
 endef
```
Comment (nikhil-zinjurde-imgtec) [inline motion-led-controller/Makefile:49]: file names in motion-led-controller github repo also need to be changed to help verify this PR

Comment (abhijit-mahajani-imgtec) [inline motion-led-controller/Makefile:49]: its fixed in https://github.com/CreatorKit/motion-led-controller/pull/4

**Comment by abhijit-mahajani-imgtec** [comment]
Approved

**Comment by abhijit-mahajani-imgtec** [comment]
Verified

**Pull request merged**